packages/store-ui/src/atoms/Bullets/Bullets.tsx

```diff
@@ -0,0 +1,44 @@
+import type { MouseEvent } from 'react'
+import React from 'react'
+
+import Button from '../Button'
+
+export interface BulletsProps {
+  totalQuantity: number
+  activeBullet: number
+  onClick: (e: MouseEvent, bulletIdx: number) => void
+  testId?: string
+}
+
+function Bullets({
+  totalQuantity,
+  activeBullet,
+  onClick,
+  testId = 'store-bullets',
+}: BulletsProps) {
+  const bulletIndexes = [...new Array(totalQuantity).keys()]
+
+  return (
+    <ol data-store-bullets data-testid={testId}>
+      {bulletIndexes.map((idx) => {
+        const isActive = activeBullet === idx
+
+        return (
+          <li
+            key={idx}
+            data-testid="bullet-item"
```

Comment:
Mixed feelings in here. Why can the dev choose the external testeId (store-bullets) but no be able to set the testId in here?